### DISCUSSION TITLE
Needs some info on how exceptions are handled here

### DISCUSSION THREAD

**Discussion by ajayd-san** [encode/httpx#1660] [Potential Issue]
hey, I want to contribute to this project so I picked the issue #1644. 
I narrowed down the search to where exceptions are raised, i.e `map_httpcore_exception` in `default.py`, now I cant find where we manipulate error messages, like the part in the traceback - 
```
httpx.UnsupportedProtocol: Unsupported URL protocol ''
```
Any pointers what I should be looking for ?

**Comment by lovelydinosaur** [comment]
I'll go answer against that ticket.
